modules/survey_module/jsx/DirectEntry.js

```diff
@@ -0,0 +1,384 @@
+/**
+ * This file contains React component for Direct Data Entry
+ *
+ * @author Jordan Stirling (StiringApps ltd.)
+ * @version 0.0.1
+ *
+ */
+
+/**
+ * Panel component
+ * Wraps children in a collapsible bootstrap panel
+ */
+
+import Page from './DirectEntryForm';
+
+/*eslint-disable */
+class DirectEntry extends React.Component {
```

Comment:
The indentation should be 2 spaces

Reply:
2 spaces is only used to reduce size of javascript files. Since this is compiled into a minified file I feel like 4 spaces is better suited as it aligns with our standards for php along with making this file easier to read

Reply:
But 4 spaces do not align with our JSX standard and make files harder to read